packages/components/pie-button/package.json

```diff
@@ -10,7 +10,11 @@
     "build": "run -T vite build",
     "watch": "run -T vite build --watch",
     "dev": "run -T vite",
-    "test": "echo \"Error: no test specified\" && exit 0"
+    "test": "echo \"Error: no test specified\" && exit 0",
+    "test:browsers": "npx playwright test -c ./playwright-lit.config.ts",
+    "test:browsers:ci": "yarn test:browsers",
+    "test:visual": "run -T cross-env-shell PERCY_TOKEN=${PERCY_TOKEN_PIE_BUTTON} percy exec --allowed-hostname cloudfront.net -- npx playwright test -c ./playwright-lit-visual.config.ts",
```

Comment:
Annoyingly, I can only get the tests to work if the playwright config for lit lives in the `pie-button` root (unlike monorepo root for pie-docs tests).

I _think_ this is because Playwright uses Vite to build pie-button before the test run, it's unable to find the pie-button`src` if living futher up the file tree.

Will have to come up with a better long-term solution for this.

Reply:
Is it worth creating a ticket to investigate this, just so we have it down somewhere in Test Excellence (or somewhere similar)